K2Bridge.Tests.UnitTests/KustoDAL/ElasticResponseTests.cs

```diff
@@ -103,7 +103,7 @@ public string MapRowsToHits_WhenMultiWordDifferentCapsCapialLettersInQueryString
         }
 
         [TestCase(ExpectedResult =
-            "{\"responses\":[{\"aggregations\":{\"2\":{\"buckets\":[]}},\"took\":0,\"timed_out\":false,\"_shards\":{\"total\":1,\"successful\":1,\"skipped\":0,\"failed\":0},\"hits\":{\"total\":{\"relation\":\"eq\",\"value\":0},\"max_score\":null,\"hits\":[]},\"status\":200}],\"took\":0}")]
+            "{\"responses\":[{\"aggregations\":{\"\":{\"buckets\":[]}},\"took\":0,\"timed_out\":false,\"_shards\":{\"total\":1,\"successful\":1,\"skipped\":0,\"failed\":0},\"hits\":{\"total\":{\"relation\":\"eq\",\"value\":0},\"max_score\":null,\"hits\":[]},\"status\":200}],\"took\":0}")]
```

Comment:
an empty key looks wrong. I guess the input value should be changed and then we should expect to have a value in this key

Reply:
Added a bug to manage this point later.

Reply:
can you add the link?

Reply:
Link to the issue added to code.